### DISCUSSION TITLE
Quieo aprender.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#142161] [New to GitHub] [answered]
### Select Topic Area

General

### Body

la verdad es que necesito de alguien aue me ayude a progrsar en la comprension y utilizacion de github, Desde ya al que asuma el compromiso realmente se lo agradeceria si es que pueden o tiene ganas, Desde ya muchas Graciasy Dios los Benciga a todos con su sabiduria,

**Comment by ghostinhershell** [comment]
Hi @Willy1997Lora1968, thanks for participating!

Our help docs are available in Spanish: https://docs.github.com/es/get-started. I recommend starting there. Take a look and please feel free to post again if you have any additional questions.

**Reply by ghost** [reply, marked as answer]
Desde ya te agradezco el tiempo que me has dedicado ya que desconozco muchas acciones de github ya que no tengo con quien compartir mis conocimientos ya sean pocos o muchos, yo tengo una pagina en la web que la hice como un borrador ingresas  con la url : leisrl.com , significa muchísimo para mi que me hayas contestados cosa que no me han pasado con otras comunidades. Desde ya te estoy muy agradecido te deseo lo mejor y que todos tus sueños se te hagan realidad en tu vida, gracias saludos cordiales Ernesto

**Comment by AWilly0897** [comment]
Ha pasado un tiempo me recomiendan no hacer uso del nombre, será que activa las propiedades del sistema, se ha progresado mucho en menos dos años a que se deben los parches. voy en busca de lo que se encuentra y no se ve, que será , yo también tengo mis pensamientos difusos sobre como funcionan las cosas te agradezco. lo principal en esto es saber ingles y leer la documentación . dime como hago ubicar las palabras como corresponden en ingles. gracias discúlpame por no haberme comunicado de forma inmediata gracias por tu ayuda, espero que lo que haya echo algo te haya servido.
